browser/ui/BUILD.gn

```diff
@@ -153,6 +153,8 @@ source_set("ui") {
       "webui/settings/brave_search_engines_handler.h",
       "webui/settings/brave_settings_localized_strings_provider.cc",
       "webui/settings/brave_settings_localized_strings_provider.h",
+      "webui/settings/brave_settings_secure_dns_handler.cc",
```

Comment:
this should also be in the block with enable_brave_vpn and is_win https://github.com/brave/brave-core/pull/15989/files#diff-7ce4cface9f839816f0fc02bad0e04aa80284b048ef0d8f0876547e6e37e38b6R297

Reply:
done in https://github.com/brave/brave-core/pull/15989/commits/d3dcf2894fc6da7caea897905f2a167bc453e3c5